.circleci/config.yml

```diff
@@ -24,16 +24,6 @@ STUFF:
         - /\//
 
 jobs:
-  lint:
-    working_directory: /app
-    docker:
-      - *python_image
-    steps:
-      - checkout
-      - run: pip install yapf==0.16.1 && pip install flake8==3.5.0 && pip install pytest==3.6.3
-      - run: yapf --diff --recursive ./
-      - run: flake8 .
```

Comment:
looks like this overlaps with #99, which i call out that linting tests shouldn't be removed from our CI runs.

Reply:
@globau thanks for the feedback -- my goal is to merge #99 before this one and then rebase this branch with the updated master. In terms of removing linting from CI - please see comment on #99.